libtextsecure/sendmessage.js

```diff
@@ -453,6 +453,30 @@ MessageSender.prototype = {
 
     return libsession.getMessageQueue().sendSyncMessage(openGroupsSyncMessage);
   },
+  async sendBlockedListSyncMessage() {
+    // If we havn't got a primaryDeviceKey then we are in the middle of pairing
+    // primaryDevicePubKey is set to our own number if we are the master device
+    const primaryDeviceKey = window.storage.get('primaryDevicePubKey');
+    if (!primaryDeviceKey) {
+      return Promise.resolve();
+    }
+    const convos = window.getConversations().models;
+
+    const conversations = Array.isArray(convos) ? convos : [convos];
+
+    const blockedConvos = await libsession.Utils.SyncMessageUtils.filterBlockedNumbers(
```

Comment:
couldn't we use `BlockedNumberController.getBlockedNumbers()`?
It has a separate block list for groups too

Reply:
We don't share blocked groups for now as mobile is not ready to handle them for now.
I will use `getBlockedNumber()`

Reply:
fixed